src/main/gov/nasa/jpf/solver/SolverWrapper.java

```diff
@@ -26,6 +26,10 @@
   public SolverWrapper(ConstraintSolver back) {
     this.back = back;
   }
+
```

Comment:
I am actually not sure, but it is legacy code, that has been in the repository prior to my project and I have not changed the existing behavior.